stable_audio_tools/interface/gradio.py

```diff
@@ -665,7 +667,15 @@ def create_ui(model_config_path=None, ckpt_path=None, pretrained_name=None, pret
     else:
         model_config = None
 
-    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    if platform.system() == "Darwin":
```

Comment:
I believe using the `torch.backend.mps.is_available()` call is more appropriate here, docs here: https://pytorch.org/docs/stable/notes/mps.html (not sure what PyTorch version is needed for this)

Reply:
Sure. Amended to have a robust check for that.